src/irmin-pack/inode.ml

```diff
@@ -214,19 +214,18 @@ struct
         let open Irmin.Type in
         variant "Compress.value"
           (fun contents_ii
-               contents_x_ii
-               node_ii
-               contents_id
-               contents_x_id
-               node_id
-               contents_di
-               contents_x_di
-               node_di
-               contents_dd
-               contents_x_dd
-               node_dd
-               ->
-          function
+          contents_x_ii
+          node_ii
+          contents_id
+          contents_x_id
+          node_id
+          contents_di
+          contents_x_di
+          node_di
+          contents_dd
+          contents_x_dd
+          node_dd
+          -> function
```

Comment:
This one looks broken to me. At the least, I would expect the arguments to be left-aligned _inside_ of the left paren, but I'm not sure why this indentation has changed at all.